src/test/java/edu/oakland/test/display01/SatelliteSignalCheckRequestUnitTests.java

```diff
@@ -9,7 +9,7 @@
 import org.junit.jupiter.api.Test;
 
 @DisplayName("Sattelite Signal Check Request Tests")
```

Comment:
This was the part that needed to be updated.
```suggestion
@DisplayName("SatelliteSignalCheckRequest Unit Tests")
```